python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -1005,8 +1014,8 @@ def errFunc(p, x, y):
                 if self.config.binSize > 1:
                     bounds = self._boundsForAstier(parsIniPtc)
                 else:
-                    bounds = self._boundsForAstier(parsIniPtc, lowers=[-1e-4, 0.5, -2000],
```

Comment:
How necessary are these bounds for the convergence of this initial fit? In any case, I'm just wondering if even the new values (min=0.2, and max=3.0) will give us trouble in the future. Or if this code is used for other cameras with different settings (I think DECam has gains of 4, so I'm not even sure at this point how this has worked in the past for DECam...)

Reply:
I don't think anybody has ever run this on DECam.  I'll change this to a bigger range.

Reply:
I did!